code-challenges.js

```diff
@@ -31,38 +58,101 @@ const temperature2 = 350
+    return `"${temperature1} is below boiling point"`
+  } 
+    else if(temperature2 >= 212) {
+    return `"${temperature2} is above boiling point"`
+  } 
+    else if(temperature3 === 212) {
+    return `"${temperature3} is at boiling point"`
+  } 
+    else {
+    return "something is wrong"
+  }
+}
+console.log(givenNum(temperature1))
+console.log(givenNum(temperature2))
+console.log(givenNum(temperature3))
+//I AM STUCK HERE-----I can't get temp3 to log at boiling point
+
+
 // --------------------2) Create the code that will combine the two arrays and return the length using the test variables provided below.
 
 // Pseudo code:
 
+// input: combine two arrays
+// output: arrays are added together to equal length
+// create a parameter that takes two arrays of data types
+// combine length of arrays
+// return the combined length
+
 const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
 const padres1998WorldSeriesRuns = [6, 3, 5, 3]
 // Expected output: 9
 
+const combinedRuns = [padres1984WorldSeriesRuns.concat(padres1998WorldSeriesRuns)]
+  let combinedArrays = padres1984WorldSeriesRuns.concat(padres1998WorldSeriesRuns)
+
+console.log(combinedArrays.length)
+
+
 // --------------------3) Create the code that will reverse the letters of a string using the test variable provided below.
 
 // Pseudo code:
 
+// input: 1 string changed into an array
+// output: 1 array reversed letters
+// create a string and change it into an array to reverse the letters
+// Use three built-in methods
+// returns the letters and numbers joined and reversed 
+
 const currentCohort = "Bravo 2023"
 // Expected output: "3202 ovarB"
 
+console.log(currentCohort.split("").reverse(" ").join(""))
+
+
 // --------------------4) Create the code that will return the last index of the given value from the array using the test variables provided below.
 
 // Pseudo code:
 
+// input: 2 variables
+// output: last index of given value from array
+// create two variables with string parameters and array
+// take length of given values in array number
+// return last index of given number
+
 const myNumbers = [13, 34, 42, 5, 5, 10, 27, 42, 10]
 const givenValue1 = 42
 // Expected output: 7
 
 const givenValue2 = 10
 // Expected output: 8
 
+console.log(myNumbers.length -2)
+console.log(myNumbers.length -1)
+
+
 // --------------------5) Create the code that will sort all the numbers in an array from largest to smallest. Use the test variables provided below.
 
 // Pseudo code:
 
+// input: 2 arrays built-in method
+// output: 2 arrays largest to smallest
+// create an array with a built-in mutator method
+// sort the order of numbers
+// return reverse to largest to smallest
+
 const sanDiegoSummerTemperatures = [79, 80, 72, 73, 82, 77, 76]
 // Expected output: [82, 80, 79, 77, 76, 73, 72]
 
 const sanDiegoWinterTemperatures = [59, 68, 62, 59, 66, 67, 66]
 // Expected output: [68, 67, 66, 66, 62, 59, 59]
+
+sanDiegoSummerTemperatures.sort((a, b) => a - b)
```

Comment:
What is the purpose of a and b in this sort method? Do we need a reverse method as well?